infra/base-images/base-builder/compile_honggfuzz

```diff
@@ -17,8 +17,8 @@
 
 echo "Skipping compilation; using precompiled honggfuzz"
 
-cp $PRECOMPILED_DIR/honggfuzz.a $LIB_FUZZING_ENGINE
-cp $PRECOMPILED_DIR/honggfuzz $OUT/
+cp $PRECOMPILED_DIR/honggfuzz/honggfuzz.a $LIB_FUZZING_ENGINE
+cp $PRECOMPILED_DIR/honggfuzz/honggfuzz $OUT/
```

Comment:
Is this change correct, i see 
cp honggfuzz $PRECOMPILED_DIR
so path should be $PRECOMPILED_DIR/honggfuzz ? and not $PRECOMPILED_DIR/honggfuzz/honggfuzz

Reply:
Yeah fixed. Sorry. 
Why do do this precompiled_dir anyway? Is it to avoid users breaking things by messing with the honggfuzz dir?

Reply:
can just remove this hack, not use precompiled dir at all.